### PR TITLE
Fix typo: the past tense of shutdown is shutdown, not shutdowned

### PR DESCRIPTION
**Pull request by GabrielBrascher**
## Description
Fixes: #3655

## Types of changes

- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

File: api/src/main/java/com/cloud/vm/VirtualMachine.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
         Migrating(true, "VM is being migrated.  host id holds to from host"),
         Error(false, "VM is in error"),
         Unknown(false, "VM state is unknown."),
-        Shutdowned(false, "VM is shutdowned from inside");
+        Shutdown(false, "VM is shutdown from inside");
```
Comment (rhtyd) [inline api/src/main/java/com/cloud/vm/VirtualMachine.java]: @GabrielBrascher Add database upgrade path to update any VMs in the old state to the new state name. (just a rare edge case).

Comment (GabrielBrascher) [reply api/src/main/java/com/cloud/vm/VirtualMachine.java]: @rhtyd added a database upgrade schema from 4.13.0.0 to 4.13.1.0 upgrading VMs from state `Shutdowned` to `Shutdown`.

Comment (GabrielBrascher) [reply api/src/main/java/com/cloud/vm/VirtualMachine.java]: @rhtyd does it look like a 4.13.1 fix? I had mapped it to 4.13.1.0, but I can target it to master/4.14.0.0; not really a bug fix, more likely an enhancement.

**Comment by GabrielBrascher** [comment]
@blueorangutan package

**Comment by GabrielBrascher** [comment]
@DaanHoogland Do you mean adding this change on a release notes?

**Comment by GabrielBrascher** [comment]
@rhtyd are you ok with this PR? Database upgrade has been covered.

**Comment by rhtyd** [comment]
Hi @GabrielBrascher let me review this and get back to you soon

**Comment by rhtyd** [comment]
This could be considered major change @GabrielBrascher - can you move this to 4.13->4.14 upgrade path on master?

**Comment by GabrielBrascher** [comment]
No problem @rhtyd, I will move it to 4.14

**Comment by GabrielBrascher** [comment]
@rhtyd code has been updated, ported to master and added SQL UPDATE into `schema-41300to41400.sql`.

**Comment by GabrielBrascher** [comment]
Fixed conflict on `schema-41300to41400.sql`; now the branch has no conflicts with the master branch.

**Comment by GabrielBrascher** [comment]
@andrijapanicsb this one looks like a simple "fix" and has 2 LGTMs aiming it for 4.14. Can you please take a look?

@rhtyd thanks for the review, are you ok with the code the way that it is after updating the upgrade path?

**Comment by andrijapanicsb** [comment]
Looks OK @GabrielBrascher and the schema updated path as well.

**Comment by andrijapanicsb** [comment]
@blueorangutan package

**Comment by andrijapanicsb** [comment]
@blueorangutan test

**Comment by andrijapanicsb** [comment]
@blueorangutan test

**Comment by andrijapanicsb** [comment]
Let there be grammar!

**Pull request merged**